ui/src/reducers/user.js

```diff
@@ -46,6 +52,25 @@ const userReducer = (state = initialState, action) => {
         .set('data', fromJS(action.payload.data));
     case USER_SET_PREFERRED_CITE_FORMAT:
       return state.set('preferredCiteFormat', action.payload.format);
+    case USER_SET_ORCID_PUSH_SETTING_REQUEST:
+      return state
+        .set('isUpdatingOrcidPushSetting', true)
+        .set(
+          'updateOrcidPushSettingError',
+          initialState.get('updateOrcidPushSettingError')
+        );
+    case USER_SET_ORCID_PUSH_SETTING_ERROR:
+      return state
+        .set('isUpdatingOrcidPushSetting', false)
```

Comment:
```suggestion
        .set(initialState.isUpdatingOrcidPushSetting)
``` ?

Reply:
I think ti's better to be explict that we want it to set false after an error.

Reply:
we have this in other places though, just for consistency but that's fine

Reply:
well we shouldn't, if you check all `loading` states, they are handled like that (explicitly)